### DISCUSSION TITLE
Fedora 40 Server Edition experience (satisfactory server)

### DISCUSSION THREAD

**Discussion by demianph** [GameServerManagers/LinuxGSM#4647] [General]
Just set up a satisfactory server in a Fedora 40 Server Edition system.
Ran into a few issues.
Initially following the instructions gave a LOT of errors (forgot to capture screen session output, but I can uninstall and re-create).
I specifically added the sfserver user to wheel group so it would have system admin access (seemed to work).
It could not run steamcmd for some reason.
looked through the .steam sub directory tree and saw only a 32bit dir under steamcmd.
I fixed this by (as root) installing steam from RPMFusion.
Then the ./sfserver install finally ran correctly and installed the server.
After this there was now a 64bit dir under steamcmd as well.
One other message specific to LGSM that I hope will be addressed in future.

Checking Satisfactory Dependencies
=================================
checking GitHub fedora-40.csv...ERROR
checking Bitbucket fedora-40.csv...FAIL
Warning! LinuxGSM dependency checking currently unavailable for Fedora Linux 40 (Server Edition).

Otherwise it is working fine.

**Comment by h3o66** [comment]
general:
- steam / steamcmd is 32bit
- as fedora is not offical supported , we do not have dependencies in the repo 
- we would need to test every half year all servers on the new fedora version as they have a release cycle this way.
- if you want to use a rpm based system and your hardware is not bleeding edge, you can use almalinux 9 or rockylinux 9
